source/Playnite.DesktopApp/Windows/GameEditWindow.xaml

```diff
@@ -546,29 +546,51 @@
                     <CheckBox Grid.Column="2" Grid.Row="7" Margin="3"
                               IsChecked="{Binding EditingGame.Favorite}" />
 
-                    <Canvas Height="20" Grid.Column="0" Grid.Row="8" />
+                    <!-- HDR -->
+                    <TextBlock Grid.Column="1" Grid.Row="8" Text="{DynamicResource LOCGameHdrTitle}" />

```

Comment:
You need to add new RowDefinition for this new option.

Reply:
Done (and surprised I missed this the first time)